env/srw_common

```diff
@@ -8,7 +8,7 @@ module load hdf5/1.10.6
 module load netcdf/4.7.4
 module load pio/2.5.2
 module load esmf/8.3.0b09
-module load fms/2022.01
+module load fms/2022.04
```

Comment:
wcoss2 does not use the env/srw_common module?

Reply:
Yes. current, wcoss2 does not use srw_common.